manim/mobject/mobject.py

```diff
@@ -1712,6 +1714,7 @@ def stretch_to_fit_depth(self, depth: float, **kwargs) -> Self:
 
         return self.rescale_to_fit(depth, 2, stretch=True, **kwargs)
 
+    @internal
```

Comment:
why?

Reply:
To be honest, I don't remember. However, I think that it's more common to use `move_to` or `set_(x|y|z)` as opposed to this method.
This PR was mostly an introduction of concept, so I'm happy to remove it if you disagree.